### DISCUSSION TITLE
How to get CtranslaterX on subtitle edit

### DISCUSSION THREAD

**Discussion by mohamed136** [SubtitleEdit/subtitleedit#7082] [General]
It only shows this when I chose CtranslaterX on subtitle edit

<img width="673" alt="Screenshot_5" src="https://github.com/SubtitleEdit/subtitleedit/assets/42202937/a2322f19-8d05-4a35-b145-0f73961b431b">

**Comment by faon-92** [comment]
see the screen capture
<img width="404" alt="SubtitleEdit_ExCVvB07kQ" src="https://github.com/SubtitleEdit/subtitleedit/assets/127494540/5852ea05-342b-4dc7-800a-4ea4100498dd">

**Reply by mohamed136** [reply]
I already figured that out, but the next problem i have is subtitles synchronize. How to make 2 subtitles have a good timing?

**Comment by StronOne** [comment]
Hi guys, could you please explain how you got this work?


Thanks in advance!
